src/shared/install.c

```diff
@@ -2886,10 +2941,20 @@ static int query_presets(const char *name, const Presets presets) {
                 log_debug("Preset files don't specify rule for %s. Enabling.", name);
                 return 1;
         case PRESET_ENABLE:
-                log_debug("Preset files say enable %s.", name);
+                if (instance_name_list && *instance_name_list)
+                        STRV_FOREACH(s, *instance_name_list) {
+                                log_debug ("Preset files say enable %s.", *s);
+                        }
+                else
+                        log_debug("Preset files say enable %s.", name);
                 return 1;
         case PRESET_DISABLE:
-                log_debug("Preset files say disable %s.", name);
+                if (instance_name_list && *instance_name_list)
+                        STRV_FOREACH(s, *instance_name_list) {
+                                log_debug ("Preset files say disable %s.", *s);
```

Comment:
No {}, no space between func name and paren, see CODING_STYLE.

So... here you use `if (instance_name_list && *instance_name_list)` as the condition, but below just `if (instance_name_list)`. It should be the same in both cases. `if (instance_name_list)` is enough.

Reply:
Hi, yea, I did not notice the {} there, thanks for pointing it out.

For the if statement, I felt the usage here is slightly different to the usage in other condition. The reason why I checked both `instance_name_list` and `*instance_name_list` here is because `instance_name_list` is a triple pointer passed into the function (to assign a `char **` to it) 

Would changes still be needed if that's the case?

Reply:
Ah, OK.